src/main/java/org/alliancegenome/curation_api/services/validation/dto/associations/EvidenceAssociationDTOValidator.java

```diff
@@ -12,17 +11,17 @@
 import org.alliancegenome.curation_api.services.validation.dto.base.BaseDTOValidator;
 import org.apache.commons.collections.CollectionUtils;
 
-import jakarta.enterprise.context.RequestScoped;
-import jakarta.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
 
 @RequestScoped
 public class EvidenceAssociationDTOValidator extends BaseDTOValidator {
 
-	@Inject InformationContentEntityService informationContentEntityService;
+	@Inject
+	InformationContentEntityService informationContentEntityService;
 
 	public <E extends EvidenceAssociation, D extends EvidenceAssociationDTO> ObjectResponse<E> validateEvidenceAssociationDTO(E association, D dto) {
 		ObjectResponse<E> assocResponse = validateAuditedObjectDTO(association, dto);
-		association = assocResponse.getEntity();
```

Comment:
why has this been removed?

Reply:
Replaced by my PR

Reply:
It's not needed as the changes are done on that object as it is passed into the method.